src/style/breakpoints.scss

```diff
@@ -1,4 +1,4 @@
-$xs: 480px;
+$xs: 400px;
```

Comment:
Why do we have this breakpoint?

Reply:
Used for really small displays aka my phone :D (iPhone SE)
Currently only hides Search and is used for Image Detail menu